icepyx/core/query.py

```diff
@@ -497,9 +513,9 @@ def CMRparams(self):
         # dictionary of optional CMR parameters
         kwargs = {}
         # temporal CMR parameters
-        if hasattr(self, "_start") and hasattr(self, "_end"):
-            kwargs["start"] = self._start
-            kwargs["end"] = self._end
+        if hasattr(self, "_temporal"):
+            kwargs["start"] = self._temporal.start
+            kwargs["end"] = self._temporal.end
```

Comment:
per our ongoing conversation about private variables - should these be `self._temporal._start` instead of `self._temporal.start`?